### PR TITLE
added example for classes as values

### PR DESCRIPTION
**Pull request by s-bessey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -631,3 +631,51 @@ animals:
...
+shearing:
+  month:
+    type: enum
+    description: what season to shear animals
+    class: month # This enum will only allow a value from the "month" class
+    default: april
+  to_shear:
+    type: array
+    description: which animals need to be sheared
+    class: animal
```
Comment (mcmcgrath13) [inline README.md:676]: add `animal` to the `classes`?

File: README.md [hunk 2]
```diff
@@ -631,3 +631,51 @@ animals:
...
+    sheep:
+      goes: baaaah
+      is_mammal: true
+    cat:
+      goes: meow
+      is_mammal: true
+    llama:
+      goes: spits
+      is_mammaL: true
+  ## we will only be using "spring," which is the default month, so no definition needed!
```
Comment (mcmcgrath13) [inline README.md]: should this say april instead of spring?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]
🐑

File: README.md [hunk 3]
```diff
@@ -635,6 +635,22 @@ The `classes` key as a root key of the parameter definitions takes on special me
...
+    fields:
+      goes:
+        type: any
+        description: What noise does the animal make?
+        default: oink
+      is_mammal:
+        type: boolean
+        description: Is this animal a mammal
+        default: false
+  default:
```
Comment (mcmcgrath13) [inline README.md]: the `default` section looks off an indent level to me

Comment (mcmcgrath13) [reply README.md]: should sheep be in our default animals?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]
🦙

File: README.md (hunk 3 above)
Comment (s-bessey) [reply README.md:650]: Yeah, I guess just making the animals used all defaults and not re-define any animals would work the best probably

**Review by mcmcgrath13 — APPROVED** [review]

**Pull request merged**